pkg/client/clientset/versioned/clientset.go

```diff
@@ -19,7 +19,11 @@ package versioned
 import (
 	"fmt"
 
+	kubeflowv1 "github.com/kubeflow/training-operator/pkg/client/clientset/versioned/typed/mpi/v1"
+	kubeflowv1 "github.com/kubeflow/training-operator/pkg/client/clientset/versioned/typed/mxnet/v1"
+	kubeflowv1 "github.com/kubeflow/training-operator/pkg/client/clientset/versioned/typed/pytorch/v1"
```

Comment:
@gaocegege This is the issue I mentioned with the imports. I'm not sure if I missed something or I should manually fix them?

Reply:
Oh, get it.